app/src/main/res/layout/comment_normal_vote_buttons.xml

```diff
@@ -1,23 +1,47 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                xmlns:app="http://schemas.android.com/apk/res-auto"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="8dp"
-                android:layout_marginRight="8dp"
-                android:paddingTop="4dp"
-                android:paddingBottom="4dp">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Wäre super, wenn du auf das LinearLayout noch verzeichten könntest. Du kannst das CommentSpacerView ja auch einfach als Kind in das RelativeLayout aufnehmen.